webapp/channels/src/components/admin_console/team_channel_settings/convert_and_remove_confirm_modal.tsx

```diff
@@ -43,34 +41,48 @@ type Props = {
 };
 
 const ConvertAndRemoveConfirmModal = ({show, onConfirm, onCancel, displayName, toPublic, removeAmount}: Props) => {
-    const toPublicMsg = 'When you convert **{displayName}** to a public channel, history and membership are preserved. Public channels are discoverable and can by joined by users on the system without invitation.  \n \nAre you sure you want to convert **{displayName}** to a public channel?';
-    const toPrivateMsg = 'When you convert **{displayName}** to a private channel, history and membership are preserved. Publicly shared files remain accessible to anyone with the link. Membership in a private channel is by invitation only.  \n \nAre you sure you want to convert **{displayName}** to a private channel?';
-    const convertMessageId = toPublic ? t('admin.team_channel_settings.convertConfirmModal.toPublicMessage') : t('admin.team_channel_settings.convertConfirmModal.toPrivateMessage');
-
-    const toPublicTitle = 'Convert channel to public and remove {amount, number} {amount, plural, one {user} other {users}}?';
-    const toPrivateTitle = 'Convert channel to private and remove {amount, number} {amount, plural, one {user} other {users}}?';
-    const titleId = toPublic ? t('admin.team_channel_settings.convertAndRemoveConfirmModal.toPublicTitle') : t('admin.team_channel_settings.convertAndRemoveConfirmModal.toPrivateTitle');
-
-    const toPublicConfirmMsg = 'Yes, convert channel to public and remove {amount, number} {amount, plural, one {user} other {users}}';
-    const toPrivateConfirmMsg = 'Yes, convert channel to private and remove {amount, number} {amount, plural, one {user} other {users}}';
-    const confirmMsgId = toPublic ? t('admin.team_channel_settings.convertAndRemoveConfirmModal.toPublicConfirm') : t('admin.team_channel_settings.convertAndRemoveConfirmModal.toPrivateConfirm');
-
-    const title = (
-        <FormattedMessage
-            id={titleId}
-            defaultMessage={toPublic ? toPublicTitle : toPrivateTitle}
-            values={{displayName, amount: removeAmount}}
-        />
-    );
+    let title;
```

Comment:
0/5 to make this file read better.

If we define the messages outside of the component (i.e. `const messages = defineMessages({toPublicTile: ..., toPublicMessage: ..., toPrivateTitle: ..., toPrivateMessage: ..., ...});`) we may keep things simpler as:

```
const tile = toPublic ? messages.toPublicTtile : messages.toPrivateTitle;
const convertMessage = toPublic ? messages.toPublicMessage : messages.toPrivateMessage;
const confirmMessage = toPublic ? messages.toPublicConfirm : messages.toPrivateConfirm;
...
    <FormattedMessage {...title} values={{amount: removeAmount}} />
...
    <FormattedMessage {...convertMessage} values={{displayName}} />
```